src/main/java/com/rental/carshowroom/config/ApplicationConfig.java

```diff
@@ -0,0 +1,32 @@
+package com.rental.carshowroom.config;
+
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.mail.javamail.JavaMailSender;
+import org.springframework.mail.javamail.JavaMailSenderImpl;
+
+import java.util.Properties;
+
+@Configuration
+public class ApplicationConfig {
+
+    @Bean
+    @Qualifier("myEmail")
+    public JavaMailSender javaMailSender() {
```

Comment:
email access data should be in properties, this properties file shouldn't be put into repo